src/main/webapp/app/shared/subject/subject-detail.component.ts

```diff
@@ -16,10 +17,12 @@ export class SubjectDetailComponent implements OnInit, OnDestroy {
     subject: Subject;
     private subscription: any;
     private eventSubscriber: Subscription;
+    public siteSettings$: Observable<SiteSettings>;
```

Comment:
You can merge this line and line #25 here.
```
public siteSettings$ = this.siteSettingsService.getSiteSettings();
```
And you can remove the Observable import line on top.